src/System.Management.Automation/engine/hostifaces/Connection.cs

```diff
@@ -700,6 +701,16 @@ internal System.Management.Automation.ExecutionContext ExecutionContext
             }
         }
 
+        /// <summary>
+        /// Gets the engine intrinsics.
+        /// </summary>
+        /// <exception cref="InvalidRunspaceStateException">Runspace is not opened.
+        /// </exception>
+        public EngineIntrinsics EngineIntrinsics
+        {
+            get => ExecutionContext?.EngineIntrinsics;
```

Comment:
@daxian-dbw That's a great question. I added that exception to the comments because the property I invoke (ExecutionContext) indicates that it raises that exception if the runspace is not opened. Since it was already documented that way, it didn't seem necessary to chase down through the code to verify whether or not that was true; however, since you called it out, I reviewed the code more closely and it seems that the InvalidRunspaceStateException is not raised by ExecutionContext, so I will remove it from both EngineIntrinsics and the ExecutionContext properties.